### ISSUE TITLE
Chatwoot v3.13.0 Facebook & Instagram delete message

### ISSUE THREAD

**Issue by evelyn-vnx** [chatwoot/chatwoot#10338]
### Describe the bug

I have upgraded to version 3.13.0 and the error persists when deleting messages (audio, video, image, text). 
1. In case of audios that I delete from Instagram or Facebook website they continue to be displayed and the message ‘This message has been deleted’ does not appear. No matter how much I refresh the Chatwoot, the audio clip is still displayed.
1.1 In case of audio clips that I delete from Instagram mobile it only goes to duration 0:00, which I take to mean that the audio was deleted, however the graphic part of the audio clip continues to be displayed and the message ‘This message has been deleted’ does not appear. 
***I am attaching an image of this.

![image](https://github.com/user-attachments/assets/e112da96-9668-422c-be26-d3ae13df9566)

2. In the case of texts, photos and videos that I send from the Facebook or Instagram website, I delete them but they are still displayed on Chatwoot. 
***I am attaching an image of my last message sent from the web which was on Oct 21 02:07 PM, a ‘Hello’ text. I then sent others but deleted them as a test and in Chatwoot they continue to show.

![image](https://github.com/user-attachments/assets/e8cf4de1-5016-4a62-ae7a-fdb7fb728011)

![image](https://github.com/user-attachments/assets/73f77431-6d43-42ff-814f-492f16612462)

### To Reproduce

1. Log in to Instagram, find your account, go to messages and send a text, photo, audio or video message.
2. Delete message from IG or FB website.
3. Go back to Chatwoot and validate that the message continues, refresh the page and the messages will not have been deleted.

### Environment

app.chatwoot.com

### Cloud Provider

None

### Platform

None

### Additional context

I submitted an issue for a problem related to this, it was apparently solved with this [https://github.com/chatwoot/chatwoot/pull/9996], however when updating I see that it still hasn't been fixed properly.

**Comment by muhsin-k** [comment]
@evelyn-vnx Are you sure you want to delete the message from the mobile app? Please note that this action may not work in browsers.

**Comment by evelyn-vnx** [comment]
@muhsin-k As I mentioned, text messages, images and video from the web are not deleted, that much is clear to me. However, the audio message is not deleted correctly from the mobile application, the audio clip remains in view and I could only notice that the duration goes to 0:00, but the message ‘This message has been deleted’ does not appear.

**Comment by muhsin-k** [comment]
>However, the audio message is not deleted correctly from the mobile application, the audio clip remains in view and I could only notice that the duration goes to 0:00, but the message ‘This message has been deleted’ does not appear.

Are you experiencing this issue through the Chatwoot mobile app?

**Comment by GeorgVolkov** [comment]
maybe this solution will solve your problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by evelyn-vnx** [comment]
@muhsin-k No, what I mean is that when I send an audio from the Instagram mobile app and delete it from the same Instagram mobile app, in the Chatwoot Web I get the audio clip in duration 0:00 which is incorrect because it should say ‘This message has been deleted’.

**Comment by CrimsonSunrise** [comment]
I came across this "issue" recently when my app got denied from Meta's review. On my video I didn't include the message deletion process. But that's very curious why it deletes if the user is doing it from the mobile app but it doesn't if he's using web. Does someone knows if this a Meta's issue or something from Chatwoot?

**Comment by rodrigowiethorn** [comment]
@CrimsonSunrise you were able to approve your app?

I'm facing an issue at the momento like this, but in reverse... i mean, unsend/delete into FB/IG app and appear as deleted in chatwoot is working fine.

But delete into chatwoot and also delete into instagram app is not working.

I already opened and issue to this: https://github.com/chatwoot/chatwoot/issues/11214

But would appreciate your comments regarding the approval process with Meta

**Comment by CrimsonSunrise** [comment]
> [@CrimsonSunrise](https://github.com/CrimsonSunrise) you were able to approve your app?
> 
> I'm facing an issue at the momento like this, but in reverse... i mean, unsend/delete into FB/IG app and appear as deleted in chatwoot is working fine.
> 
> But delete into chatwoot and also delete into instagram app is not working.
> 
> I already opened and issue to this: [#11214](https://github.com/chatwoot/chatwoot/issues/11214)
> 
> But would appreciate your comments regarding the approval process with Meta

Hey @rodrigowiethorn 

If I'm not mistaken, deleting a message in Instagram chat only sends the delete weebhook event if done on a mobile phone. It doesn't trigger if you delete on a desktop. That's a Facebook's thing, they know it. So what I've done by the time was screensharing my phone to my pc and recording the pc screen showing it.

**Comment by sojan-official** [comment]
@CrimsonSunrise  Thanks for sharing. 

> > [@CrimsonSunrise](https://github.com/CrimsonSunrise) you were able to approve your app?
> > I'm facing an issue at the momento like this, but in reverse... i mean, unsend/delete into FB/IG app and appear as deleted in chatwoot is working fine.
> > But delete into chatwoot and also delete into instagram app is not working.
> > I already opened and issue to this: [#11214](https://github.com/chatwoot/chatwoot/issues/11214)
> > But would appreciate your comments regarding the approval process with Meta
> 
> Hey [@rodrigowiethorn](https://github.com/rodrigowiethorn)
> 
> If I'm not mistaken, deleting a message in Instagram chat only sends the delete weebhook event if done on a mobile phone. It doesn't trigger if you delete on a desktop. That's a Facebook's thing, they know it. So what I've done by the time was screensharing my phone to my pc and recording the pc screen showing it.
